trikot-analytics/analytics/src/commonMain/kotlin/com/mirego/trikot/analytics/AnalyticsService.kt

```diff
@@ -16,6 +17,11 @@ interface AnalyticsService {
      */
     var isEnabled: Boolean
 
+    /*
+    The distinctAppId associated with the current device
+     */
+    fun distinctAppId(): Promise<String?>
```

Comment:
Does the String really need to be nullable? If it's null, shouldn't the  Promise be rejected instead?